### PR TITLE
Update channels.csv

### PR DESCRIPTION
**Pull request by riczuko**

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -29682,6 +29682,7 @@ XezerXeber.az,Xezer Xeber,,,,AZ,,,c/AZ,aze,,FALSE,,,,https://www.xezerxeber.az/,
 XFCTV.us,XFC TV,,,,US,,,c/US,eng,sports,FALSE,,,,https://watch.plex.tv/live-tv/channel/xfc-tv,https://i.imgur.com/LcfzmG0.png
 XHASDT2.us,XHAS-DT2,,LATV,,US,US-CA,San Diego,s/US-CA,eng,,FALSE,,,,,https://i.imgur.com/ScxwHYR.png
 XHBCTDT.mx,XHBC-TDT,,,Grupo Televisa,MX,,Mexicali,c/MX,spa,,FALSE,,,,http://www.televisaregional.com/mexicali/,https://i.imgur.com/0qaZtwz.png
+XEWTDT.mx,XEWT-TDT,,,Grupo Televisa,MX,,Tijuana,c/MX,spa,,FALSE,,,,https://xewt12.com/,https://static.wikia.nocookie.net/logopedia/images/1/16/XEWT_2018.png
```
Comment (freearhey) [inline data/channels.csv:29685]:
```suggestion
XEWTTDT.mx,XEWT-TDT,,,Grupo Televisa,MX,,Tijuana,c/MX,spa,,FALSE,,,,https://xewt12.com/,https://static.wikia.nocookie.net/logopedia/images/1/16/XEWT_2018.png
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by LaneSh4d0w** [comment]
Tried to fix the issue, but I had no write access on this repo, sorry.

File: data/channels.csv (hunk 1 above)
Comment (LaneSh4d0w) [inline data/channels.csv:29685]:
```suggestion
XEWTTDT.mx,XEWT-TDT,,,Grupo Televisa,MX,,Tijuana,c/MX,spa,,FALSE,,,,https://xewt12.com/,https://static.wikia.nocookie.net/logopedia/images/1/16/XEWT_2018.png

```

**Comment by freearhey** [comment]
@LaneSh4d0w Please try again. The admin panel states that you have write access to all repositories in the organization.

image

**Comment by LaneSh4d0w** [comment]
> @LaneSh4d0w Please try again. The admin panel states that you have write access to all repositories in the organization.
> 
> image
> 

I do have access to all the projects, but the users when forking the repository can refuse collaborators to access it.

Please try to make the changes using the fork and github.dev, it gives me an error.

**Comment by freearhey** [comment]
Now I see.

**Pull request closed without merging**